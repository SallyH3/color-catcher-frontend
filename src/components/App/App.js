import React, { Component } from 'react';
import './_App.scss';
import { Header } from '../Header/Header.js'

class App extends Component {
	constructor() {
		super();
		this.state = {
			colors: []
		}
	}

	render() {
		return(
			<div>
				<Header />
				<h1> hi </h1>
			</div>
		)
	}
}


export default App;