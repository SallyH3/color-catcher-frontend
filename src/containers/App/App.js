import React, { Component } from 'react';
import './_App.scss';
import { Header } from '../../components/Header/Header.js';
import Swatch from '../Swatch/Swatch.js';
import Projects from '../../components/Projects/Projects.js';
import Palettes from '../../components/Palettes/Palettes.js';

class App extends Component {
	constructor() {
		super();
		this.state = {
			codes: []
		}
	}

	//generate hex code function
	// generateCodes = () => {
	// 	let code = ''
	// 	codes.push(code)
	// }
	
	componentDidMount = () => {
		//call generateCodes()
	}

	render() {
		return(
			<div>
				<Header />
				<Swatch />
				<section className='file-holder'>
					<Projects />
					<Palettes />
				</section>
			</div>
		)
	}
}


export default App;