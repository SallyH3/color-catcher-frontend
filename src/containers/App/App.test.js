import React from 'react';
import { shallow } from 'enzyme';
import App from './App.js';

describe('App', () => {
	it('should match the snapshot', () => {
		const wrapper = shallow( <App />);
		expect(wrapper).toMatchSnapshot();
	});
});