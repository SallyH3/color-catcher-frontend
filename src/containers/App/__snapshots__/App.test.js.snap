// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot 1`] = `
<div>
  <Connect(Header) />
  <Connect(Swatch) />
  <section
    className="file-holder"
  >
    <Connect(Projects) />
    <Connect(Palettes) />
  </section>
</div>
`;
